xenia-build

```diff
@@ -850,15 +850,26 @@ class BaseBuildCommand(Command):
             ] + scheme_args + pass_args, shell=False, env=dict(os.environ))
         else:
             result = subprocess.call([
-                'make',
-                '-j' if threads == 0 else '-j%d' % threads,
-                '-Cbuild/',
-                'config=%s_linux' % (args['config']),
-            ] + pass_args + args['target'], shell=False, env=dict(os.environ))
-        print('')
-        if result != 0:
-            print('ERROR: build failed with one or more errors.')
-            return result
+                'cmake',
+                '-S build/',
+                '-B build/build_%s' % (args['config']),
+                '-DCMAKE_BUILD_TYPE=%s' % (args['config'].title()),
+                '-DCMAKE_C_COMPILER=clang',
+                '-DCMAKE_CXX_COMPILER=clang++',
```

Comment:
Does this override the CC/CXX env vars?

Reply:
Yeah, I guess so. But it will not compile with anything else on Linux, so it shouldn't really be an issue.

Maybe moving this to premake might be a better option.

Reply:
I fixed it in #525 so specific versions of clang/llvm can be used.